docs/readme.txt

```diff
@@ -1,172 +1,141 @@
-                wxWidgets 3.1.0 Release Notes
-                =============================
+# wxWidgets 3.1.0 Release Notes
```

Comment:
I mostly just wanted to make this template more Markdown-friendly even though it will remain a plain text file for now. I'm tempted to officially rename it to `readme.md`, but there's a lot of references to this file everywhere, so it's likely not a good idea.